### PR TITLE
bitrise-step-xcode-project-info 2.1.0

### PR DESCRIPTION
**Pull request by tadija**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by tadija** [comment]
I can't see the "details" about why this check has failed? what's the check?

**Comment by Szabadember** [comment]
(3/4) Running: Ensure all type_tags and project_type_tags are supported
 > Failed, error:
   - invalid type_tag: xcode
   - invalid type_tag: versioning

File: steps/bitrise-step-xcode-project-info/2.1.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+published_at: 2020-04-17T18:23:26.439085+02:00
+source:
+  git: https://github.com/tadija/bitrise-step-xcode-project-info.git
+  commit: afd75d12e414fbdb3907a0a5cde5b79aa299a65e
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+type_tags:
+- xcode
```
Comment (Szabadember) [inline steps/bitrise-step-xcode-project-info/2.1.0/step.yml:20]: Please check the contribution guide for valid type_tags:
https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md

> Available type_tags:
>
>access-control
>artifact-info
>installer
>deploy
>utility
>dependency
>code-sign
>build
>test
>notification

**Comment by tadija** [comment]
Thanks @Szabadember, I fixed `type_tags` in the last commit. Previous versions of this step were located in "xcode-project-info" directory but the initial commit placed it at "bitrise-step-xcode-project-info" (automatically made by stepman). This is now also fixed.

File: steps/xcode-project-info/2.1.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+title: "Xcode Project Info"
+summary: Extracts Xcode project info data to environment variables.
+description: |-
+  This step will simply extract Xcode project information (Version and Build),
+  and export them into environment variables prefixed with **XPI_**.
+
+  Afterwards, you can use these environment variables in other steps (ie. sending message on Slack).
+website: https://github.com/tadija/bitrise-step-xcode-project-info
+source_code_url: https://github.com/tadija/bitrise-step-xcode-project-info
+support_url: https://github.com/tadija/bitrise-step-xcode-project-info/issues
```
Comment (trapacska) [inline steps/xcode-project-info/2.1.0/step.yml:10]: The step.yml has some of missing fields: 
- published_at
- source:
  - git
  - commit

Please always do a step share with our stepman. Please read the share process here: 
https://devcenter.bitrise.io/contributors/sharing-steps-with-all-bitrise-users/

**Comment by trapacska** [comment]
@tadija I will close this PR as re-share will be required.
Please do the changes I mentioned and open a new PR with the new share.

Thanks!

**Comment by tadija** [comment]
Hi @trapacska, ok I will add those missing fields and make a new share. Can you please tell me what will happen if the stepman puts this step into a different directory like it already did (`bitrise-step-xcode-project-info` instead of `xcode-project-info`)? Will the step update be available normally or users would have to replace the existing step with a new one? Can I fix that directory name for stepman somehow?

**Pull request closed without merging**